GameLogic/engine/StandardFENSerializer.cs

```diff
@@ -151,7 +151,11 @@ public static string BoardToNotation(IRectangularBoard board)
                     // Set |numberEmptyCells| to zero.
                     else
                     {
-                        if (numberEmptyCells != 0) row.Add(Convert.ToChar(numberEmptyCells + 48));
+                        if (numberEmptyCells != 0)
```

Comment:
Out of curiosity, did you change it manually? Or did 'dotnet format' edit it?

Reply:
.editorconfig, yes
I prefer to highlight blocks like this